python/doc/examples/meta_modeling/general_purpose_metamodels/plot_overfitting_model_selection.py

```diff
@@ -107,31 +107,31 @@ def linearSample(xmin, xmax, npoints):
 # Compute the coefficients of the polynomial decomposition
 # --------------------------------------------------------
 #
-# Let :math:`y \in \mathbb{R}^n` be a vector of observations.
+# Let :math:`\vect{y} \in \mathbb{R}^n` be a vector of observations.
 # The polynomial model is
 #
 # .. math::
-#    P(x) = \beta_0 + \beta_1 x + ... + \beta_p x^p,
+#    P(\vect{x}) = \beta_0 + \beta_1 \vect{x} + ... + \beta_p \vect{x}^p,
 #
-# for any :math:`x\in\mathbb{R}`, where :math:`p` is the polynomial degree and :math:`\beta\in\mathbb{R}^{p+1}` is the vector of the coefficients of the model.
-# Let :math:`n` be the training sample size and let :math:`x_1,...,x_n \in \mathbb{R}` be the abscissas of the training set.
-# The design matrix :math:`X \in \mathbb{R}^{n \times (p+1)}` is
+# for any :math:`\vect{x}\in\mathbb{R}`, where :math:`p` is the polynomial degree and :math:`\beta\in\mathbb{R}^{p+1}` is the vector of the coefficients of the model.
+# Let :math:`n` be the training sample size and let :math:`\vect{x}_1,...,\vect{x}_n \in \mathbb{R}` be the abscissas of the training set.
+# The design matrix :math:`\mat{X} \in \mathbb{R}^{n \times (p+1)}` is
 #
 # .. math::
-#    x_{i,j} = x^j_i,
+#    \vect{x}_{i,j} = \vect{x}^j_i,
```

Comment:
No, the original equation was better here.

Reply:
fixed